idact/detail/allocate_resources_app/main.py

```diff
@@ -0,0 +1,144 @@
+"""This module contains the :func:`main` function for the allocating resources
+for future, see :mod:`idact.allocate_resources`.
```

Comment:
And this is not only for the future, so maybe it should be changed a little bit.